mllib/src/main/scala/org/apache/spark/mllib/api/python/PythonMLLibAPI.scala

```diff
@@ -284,6 +285,59 @@ class PythonMLLibAPI extends Serializable {
   }
 
   /**
+   * Java stub for Python mllib GaussianMixtureEM.train()
+   * Returns a list containing weights, mean and covariance of each mixture component.
+   */
+  def trainGaussianMixtureEM(
+      data: JavaRDD[Vector], 
+      k: Int, 
+      convergenceTol: Double, 
+      seed: Long, 
+      maxIterations: Int): JList[Object] = {
```

Comment:
Usually we put `seed` as the last argument. For GMM, I would suggest `k, maxIterations, convergenceTol, seed`.